docs/tutorial/intro.rst

```diff
@@ -0,0 +1,20 @@
+=============================
+Minimal PyMoskito
+=============================
+PyMoskito simulates the control loop as seen in the schematics
+below. This tutorial will focus on the part highlighted in blue,
+since these modules are essential to run the toolbox:
+
+.. image:: ../pictures/ctrl_loop_intro.png
+
+Every block in this diagram represents an abstract class or base class, that PyMoskito offers.
```

Comment:
Better: Every block in this diagram represents a configurable part of the control loop, that is implemented as a generic base class.